superset-frontend/src/api/dataset.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { SupersetClient, JsonResponse } from '@superset-ui/core';
+import rison from 'rison';
+
+export const get = async (userId: number) => {
+  const queryParams = rison.encode({
+    filters: [
+      {
+        col: 'owners',
+        opr: 'rel_m_m',
```

Comment:
We should eventually make this into an enum... I assume this means "relationship many to many"? (from https://github.com/apache/incubator-superset/blob/6019113bc6ea68f0e440d1928063db972357333c/superset-frontend/src/components/ListView/types.ts#L44)

Reply:
I added an enum for these in https://github.com/apache/incubator-superset/pull/11702/files#diff-8a9b19f5b4afdd348a7201a98d87b4c95b5bc9d5e45d26cc44913512babb24c8R103 which should be merging soon

Reply:
will create a ticket to change these out once merged